cmd/zetaclientd/debug.go

```diff
@@ -8,6 +8,13 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/zeta-chain/zetacore/zetaclient/bitcoin"
+	"github.com/zeta-chain/zetacore/zetaclient/keys"
+	"github.com/zeta-chain/zetacore/zetaclient/metrics"
+	"github.com/zeta-chain/zetacore/zetaclient/zetabridge"
+
+	"github.com/zeta-chain/zetacore/zetaclient/evm"
+
```

Comment:
```suggestion
	"github.com/zeta-chain/zetacore/zetaclient/evm"
```


We should keep imports consistent.
Same comment for other files. 
We should run `make generate` afterward to run `goimport`